### PR TITLE
feat: add documentation for imageRegistryCredentials

### PR DESCRIPTION
**Pull request by vishal-chdhry**
## Related issue 
Closes: #985 

## Proposed Changes
Adds docs for `imageRegistryCredentials`  in `verifyImage` policies that can be used to configure registry credentials per policy.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

File: content/en/docs/Writing policies/verify-images/_index.md
Comment (realshuting) [inline content/en/docs/Writing policies/verify-images/_index.md:48]:
```suggestion
The `imageRegistryCredentials.secret` specifies a list of secrets that are provided for credentials. Secrets must be in the Kyverno namespace.
```

**Review by realshuting — APPROVED** [review]

**Comment by realshuting** [comment]
Defer to @chipzoller to have the final review :)

**Review by chipzoller — APPROVED** [review]

**Pull request merged**
